k8s/demo/couchbase/README.md

```diff
@@ -0,0 +1,148 @@
+default       pvc-40e1b64f-c7ba-11e7-892e-000c29119159-rep-6cd4655d87-h7w9x    1/1       Running   0          11h
+kube-system   etcd-o-master01                                                  1/1       Running   0          3d
+kube-system   kube-apiserver-o-master01                                        1/1       Running   0          3d
+kube-system   kube-controller-manager-o-master01                               1/1       Running   0          3d
+kube-system   kube-dns-545bc4bfd4-m4ngc                                        3/3       Running   0          3d
+kube-system   kube-proxy-4ml5l                                                 1/1       Running   0          3d
+kube-system   kube-proxy-7jlpf                                                 1/1       Running   0          3d
+kube-system   kube-proxy-cxkpc                                                 1/1       Running   0          3d
+kube-system   kube-scheduler-o-master01                                        1/1       Running   0          3d
+kube-system   weave-net-ctfk4                                                  2/2       Running   0          3d
+kube-system   weave-net-dwszp                                                  2/2       Running   0          3d
+kube-system   weave-net-pzbb7                                                  2/2       Running   0          3d
+
+```
+
+Get the status of running StatefulSets:
+
+```bash
+ubuntu@kubemaster:~$ kubectl get statefulset
+NAME        DESIRED   CURRENT   AGE
+couchbase   2         2         11h
+
+```
+
+Get the status of underlying persistent volume being used by Couchbase StatefulSet:
+
+```bash
+ubuntu@kubemaster:~$ kubectl get pvc
+NAME                         STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+couchbase-data-couchbase-0   Bound     pvc-16210b06-c7ba-11e7-892e-000c29119159   5G         RWO            openebs-standard   11h
+couchbase-data-couchbase-1   Bound     pvc-40e1b64f-c7ba-11e7-892e-000c29119159   5G         RWO            openebs-standard   11h
+
+```
+
+Get the status of the services:
+
+```bash
+ubuntu@kubemaster:~kubectl get svc
+ME                                                TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+couchbase                                           ClusterIP   None             <none>        8091/TCP            11h
+couchbase-ui                                        NodePort    10.103.161.153   <none>        8091:30438/TCP      11h
+kubernetes                                          ClusterIP   10.96.0.1        <none>        443/TCP             3d
+maya-apiserver-service                              ClusterIP   10.111.26.252    <none>        5656/TCP            3d
+
+```
+
+## Launch Couchbase Server
+
+The Couchbase service YAML, creates a service of type NodePort to make the Couchbase Server available outside the cluster.
```

Comment:
creates a service of type = creates a NodePort service type
Server = server
to make the Couchbase Server = for making the Couchbase server